cmd/download.go

```diff
@@ -74,34 +74,44 @@ func get(src, dest string, mode getter.ClientMode, cleanGitFolder bool) error {
 
 	logrus.Debugf("complete url downloading: %s -> %s", src, dest)
 
+	var tempDest = dest + ".tmp"
+
 	pwd, err := os.Getwd()
 	if err != nil {
 		return err
 	}
 
 	client := &getter.Client{
 		Src:  src,
-		Dst:  dest,
+		Dst:  tempDest,
 		Pwd:  pwd,
 		Mode: mode,
 	}
 
 	logrus.Debugf("let's get %s -> %s", src, dest)
 
-	if _, err := os.Stat(dest); !os.IsNotExist(err) {
-		logrus.Infof("%s already exists! removing it", dest)
-		err = removeDir(dest)
-		if err != nil {
-			logrus.Error(err)
-			return err
-		}
-	}
-
 	humanReadableDownloadLog(src, dest)
 
 	err = client.Get()
 	if err != nil {
+		_ = removeDir(tempDest)
 		return err
+	}else{
```

Comment:
I'm pretty sure this line is breaking the linter. Should be `} else {`. Please run `go fmt` on the file :)

Reply:
holy 😄

Reply:
Done, i'm adding the target lint to the build command, since the linting is automated